wordpress-mobile/WordPress-Android
# Implement: Common styles for Dashboard cards

## Related Issue 
Closes #18375 & #18447

## Description 
This PR creates 4 base styles for the dashboard cards and updates styles of the dashboard cards to those styles. The Styles introduced can be seen in this figma link  - YSMw2nbFAgPpjZLLbYvTfT-fi-536_3771

Also fixes the View all responses don't look tappable

## Explanation of changes 
⊕ Adds Base styles in [WordPress/src/main/res/values/dashboard_card_styles.xml](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/files#diff-12d9b876fcb730b392376e60d030c0ad3e1d587eef77f40448298c92f88a072f)
⊕ [Adds: Styles for dashboard cards from the designs](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/d3559d78b483512d83756a71c4148756462c81a0)
 ⊜ [Updates: styling of site title in my site dashboard to use roboto](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/cf2d60c345cc09b254213fe152ac9307f8ec0a75)
⊜  [Updates: base style according to the material library update](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/7cfe4c582594baf64215dbc6f7f867d73a798e0f)
⊜  [Updates: styling of quick link ribbon items](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/a3ef266cae94fd1b06ef17da0637637fe228a261)

### Changes the text styles of the following Dashboard cards   
- [x] Post card 
   - [x] With Post items  
   - [x] Without Post Items 
       - [x] Create your first post card 
       - [x] Create your next post card   
   1. [Updates: Post title and subtitle](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/f89786bd7be7ab0bf1e2a7782e889bac58e5b12b)  
   2. [Fixes: Incorrect style reference in post card](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/c1f9977a036de45a156403236c15d1ca958d12b8)
- [x] Todays Stats Card - [* Updates: Stats card to inherit styles from common dashboard card style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/43b118422ce075602771795396081626fe562191)
- [x] Pages Card - [Updates: style of the pages card to use dashboard card style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/9bd46bccecdc71a8ce18c809a30ecc5ecfa9784c)
- [x] Activity Log Card - Using same styles as post card - [Updates: Post title and subtitle](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/f89786bd7be7ab0bf1e2a7782e889bac58e5b12b) 
- [x] Blogging Prompt Card - [Updates: Blaze card to inherit styles from common dashboard card style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/6ab0fbbe9432ed5bfe37a0ae6c33ade0353c7f5b)
    - [x]  [Updates: Blogging Prompt card view responses text color](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/160cfac96e26465360ca222456fd3bc86f03fad0)
    - [x] [* Updates: Blogging Prompt card to use styles from dashboard card style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/254102a84897df1b5e3d33d78282de7bc28a137c)
    - [x] The avatar size needed to be changed and since the avatar XML was being used in reader. Created a new XML just for the blogging prompt card [* Updates: Blogging Prompt card avatar size](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/a2121f95a56cc30e157b38cc1cca32a33e6f62e9)
- [x] Domain dashboard card - [* Updates: Quick start card & Domain dashboard card](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/677ee12b4b4fe9706bfc793a5ff195fc33f6a21f)
- [x] Domain Registration card - [* Updates: domain registration card subtitle style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/3ab7c2211feeafc1b3fce545936c9517f3d1a3cc)
- [x] Custom Domain Card - [* Updates: styling of custom domain card title with base style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/2dfd80ff0f647ee3f88b22b4a25918792a4091a9) 
- [x] Promote with Blaze card - [Updates: Blaze card to inherit styles from common dashboard card style](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/6ab0fbbe9432ed5bfe37a0ae6c33ade0353c7f5b)
- [x] Quick start card - [Updates: Quick start card & Domain dashboard card](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/677ee12b4b4fe9706bfc793a5ff195fc33f6a21f)
    - [x] Existing users quick start card 
    - [x] New users quick start card 
- [x] Plans card - [* Updates: dashboard_card_plans.xml to use common dashboard_card_styles](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/45e0c942bc0349a6f10cf4ef5a758fdbd99eb5f3)
- [x] Jetpack install plugin card - [Updates: the style of the Jetpack install full plugin card](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/c8d33f231700527c2d0e5992d5b115ab22dd10e3)


## To test:
- Check if the dashboard cards are shown as per design 
- Check if all the cards are shown as per the existing logic 
- Check if the cards behave as expected in 

## Regression Notes
1. Potential unintended areas of impact
- Dashboard cards UI is not correct 
- Dashboard cards are not shown 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing 

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

UI Changes testing checklist:
